### ISSUE TITLE
Upgrading from Rails 6.0.3.6 to 6.1.3.1 causes PDF download trigger browser error: ERR_RESPONSE_HEADERS_TOO_BIG

### ISSUE THREAD

**Issue by Arns** [mileszs/wicked_pdf#971]
## Issue description
I used the render PDF option for my Rails app and all works well on version 6.0.3.6. I upgraded to Rails 6.1.3.1 and now all browsers throw the following error: ERR_RESPONSE_HEADERS_TOO_BIG. Note that I've done some investigating and it appears to be linked to my usages of  <%= stylesheet_link_tag wicked_pdf_asset_base64("pdf") %>. I don't understand why that would suddenly be causing problems when upgrading to the new version of rails.

## Expected or desired behavior
PDF should download as usual.

## System specifications

wicked_pdf gem version (output of `cat Gemfile.lock | grep wicked_pdf`):
1.4.0
wkhtmltopdf version (output of `wkhtmltopdf --version`):
0.12.6
whtmltopdf [provider gem](https://rubygems.org/search?utf8=%E2%9C%93&query=wkhtmltopdf) and version if one is used:

platform/distribution and version (e.g. Windows 10 / Ubuntu 16.04 / Heroku cedar):
Doesn't apply.. error exists on Windows 10 edge/chrome, Ubuntu Chrome/Firefox

**Comment by mjtko** [comment]
Ok, I've bashed this about a bit, and it looks like it's a problem with behaviour introduced in https://github.com/rails/rails/pull/39939 and then softened in https://github.com/rails/rails/commit/c55e32ec94c80168d3fbb7eace02cb5e72034e48.

TL;DR: set this in `application.rb`:

```
config.action_view.preload_links_header = false
```

...Or do something else to prevent the massive data URLs being added to the `Link` header. Open to better ideas than entirely disabling application-wide for all URL types and will probably look into this more in due course. Feels like it's a Rails-side bug and it shouldn't include data URLs in `Link`?

**Comment by unixmonkey** [comment]
@mjtko Thank you for tracking this down! The latter commit you pointed to makes it look like you can call:
```erb
<%= javascript_include_tag('something', defer: false) %>
```

in at least the newest version of Rails.

**Comment by mjtko** [comment]
I had a quick glance at issues in the Rails repo and it looks like this PR might fix the problem (or at least hide it) in the next release of Rails. https://github.com/rails/rails/pull/42056

It sounds like that fix still might cause a lot of data might be sent that doesn't need to be sent though.

**Comment by mjtko** [comment]
@byroot Good thinking to make the last comment the workaround to save people having to read through the whole issue.

The workaround is as follows (from https://github.com/mileszs/wicked_pdf/issues/971#issuecomment-836594736 above):

Set this in application.rb:

```
config.action_view.preload_links_header = false
```

(NB. `preload_links_header` (plural) not `preload_link_header`).
